### DISCUSSION TITLE
On click show annotations

### DISCUSSION THREAD

**Discussion by Erik-Egnatosyan** [google/model-viewer#3898] [Q&A]
How can I show annotations when clicking on buttons? I have a separate button for navigation. I attached the annotations separately to the 3d model. Now I want annotations to appear when clicked. I tried the option with --min-hotspot-opacity:0; but it only works if the dot is not visible. Is there an option to show when pressed and remove if pressed, something else? I will be grateful if you help!
![image](https://user-images.githubusercontent.com/69196326/197389056-53403af9-caa6-4e55-9a97-7cd8646bd3e2.png)

**Comment by btutal** [comment]
I believe you can easily do this via CSS. Have you checked [Dimensions example](https://modelviewer.dev/examples/annotations/index.html#dimensions) ?
